### PR TITLE
PBI 37,38,39 - Create, See, and Edit Submission

### PR DESCRIPTION
**Pull request by fxdros**

### CODE REVIEW

File: src/resources/app/pages/course/Assignment.vue [hunk 1]
```diff
@@ -68,18 +78,22 @@
...
 </template>
 <script>
 import CreationLayout from "../../components/Layout/CreationLayout";
 import AppLayout from "../../components/Layout/AppLayout";
-import { getCourseAssignmentById } from "../../../api/course/assignment";
-import { courseDetail } from "../../../api/course/detail";
+import {getCourseAssignmentById} from "../../../api/course/assignment";
+import {courseDetail} from "../../../api/course/detail";
+import {uploadFile} from "../../../api/course/lessonMaterial";
+import {createSubmission, editSubmission, readSubmission} from "../../../api/course/submission";
```
Comment (AldiNFitrah) [inline src/resources/app/pages/course/Assignment.vue]: nit: spaces between curly braces

**Review by AldiNFitrah — APPROVED** [review]

File: src/resources/app/pages/course/Assignment.vue (hunk 1 above)
Comment (fxdros) [reply src/resources/app/pages/course/Assignment.vue]: sure, gonna fix it asap

File: src/resources/app/pages/course/Assignment.vue [hunk 2]
```diff
@@ -117,13 +136,61 @@ export default {
...
+        this.changeUploadText();
+      } catch (error) {
+        await this.$store.dispatch("OPEN_SNACKBAR", "Error uploading data");
+      }
+      this.isSubmitting = false;
+    },
+    async getExistingSubmission() {
+      try {
+        let response = await readSubmission(this.$props.assignmentId);
+        console.log(response)
```
Comment (IlmaMannix) [inline src/resources/app/pages/course/Assignment.vue:175]: please remove this

Comment (IlmaMannix) [inline src/resources/app/pages/course/Assignment.vue]: please remove this

**Review by IlmaMannix — APPROVED** [review]
approved 👍 , good job, thanks bas 👍👍👍

File: src/resources/app/pages/course/Assignment.vue (hunk 2 above)
Comment (fxdros) [reply src/resources/app/pages/course/Assignment.vue]: sure, on my way

**Pull request merged**